src/website/views/components/accordion/_examples.pug

```diff
@@ -0,0 +1,185 @@
+h2 Examples
+p.-text To render accordion, use the class <code>chi-accordion</code>.
+
+h3 Base
```

Comment:
@sabah1910 base example should not be bordered. Please change to default.

Reply:
@jllr Just wanted to confirm if this how it should look like without the border? 
![image](https://user-images.githubusercontent.com/58184901/93391896-1e130080-f82d-11ea-988d-7b7108589147.png)

Reply:
@sabah1910 For the example, background color should be white, not grey. Attached in the Jira card are updated screenshots of base and -portal examples.